docs/bos/queryapi/context.md

```diff
@@ -0,0 +1,348 @@
+
+:::tip
+
+The GraphQL method was previously the only way to interact with the database. Now, the [DB methods](#db) provide a more accessible functionality. The GraphQL method is better suited for more complex queries and mutations. Also, more information about GraphQL calls can be [found here](https://graphql.org/learn/queries/).
+
+:::
+
+#### Input
+
+```js
+await context.graphql(operation, variables) 
+```
+
+The operation is a string formatted to match a GraphQL query. The variables are any data objects used in the query.
+ 
+#### Example
+
+```js
+const mutationData = {
+  post: {
+    account_id: accountId,
+    block_height: blockHeight,
+    block_timestamp: blockTimestamp,
+    content: content,
+    receipt_id: receiptId,
+  },
+};
+
+
+// Call GraphQL mutation to insert a new post
+await context.graphql(
+  `mutation createPost($post: dataplatform_near_social_feed_posts_insert_input!){
+    insert_dataplatform_near_social_feed_posts_one(
+    object: $post
+    ) {
+      account_id
+      block_height
+    }
+  }`,
+  mutationData
+);
+```
+
+The above is a snippet from the social feed indexer. In this one, we have a mutation (which mutates or changes data in the database, instead of a query that merely reads) called `createPost`. The mutation name can be anything. We specify a variable `post` and execute a graphQL method, which inserts the `post` object and returns `account_id` and `block_height` from the newly inserted object. Finally, we pass in `mutationData` as the variable, which is automatically linked to `post` since it's the only field.
+
+:::tip
+
+You can find other examples of `context.graphql` in the [social_feed indexers](../tutorial/indexer-tutorials/feed-indexer.md). 
+
+:::
+
+### Set
+
+Each indexer, by default, has a table called `indexer_storage`. It has a field for `key` and `value`, functioning like a key-value store. This table can, of course, be removed from the DDL before publishing. However, it kept the `set` method as an easy way to set some value for a key in that table. This method is used in the default code to set the `height` on each invocation. 
+
+#### Input
+
+```js
+await context.set(key, value)
+```
+
+#### Example
+
+```js
+const h = block.header().height;
+await context.set("height", h);
+```
+
+### Log
+
+Code written by developers is executed on a GCP compute instance. Therefore, things like `console.log` are surfaced to the machine itself. So, to surface `console.log` statements not only to the machine but also back to the user under Indexer Status, we need to write these logs to the logging table, which is separate from the developer's schema. This actually happens in the runner. We map `console.log` statements in the developer's code to call this function instead. So, for the developer, they can simply use `console.log`, and we take care of the rest.
```

Comment:
The person reading this is going to be the developer in question. I wrote most of what I did with the audience being our team. So, some paragraph slike this one could probably be re-written to read nicely if I am a customer. 

On that note, there's no additional functionality provided to users through this method. Maybe we're better off not even mentioning it in the doc?

Reply:
Yes, I agree that we can remove the whole `Logs` section since it does not provide any helpful info for the developers reading it.

Reply:
- section removed -